src/v2/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -78,7 +78,7 @@ const RecommendedArtist: FC<
         }
       />
 
-      <Carousel mt={3} arrowHeight={HEIGHT}>
+      <Carousel my={undefined} mt={3} arrowHeight={HEIGHT}>
```

Comment:
Sort of annoying. Have to unset `my` since it has a default value. I wish styled-system was smarter with merging these values...

Reply:
Hm... I can perhaps just inline them instead of using `defaultProps`: https://github.com/styled-system/styled-system/issues/599

Reply:
OK, yeah this works better: https://github.com/artsy/palette/pull/831 will just bump palette in this PR and keep the text update because why not.